templates/idempotent-wrapper.sql

```diff
@@ -0,0 +1,8 @@
+
+-- {{filename}}
+DO
```

Comment:
Originally I figured it would make sense to have the migration and idempotent scripts structurally similar. I've since added a `RAISE LOG` line to this block, which does mandate the wrapping.